src/pages/index.js

```diff
@@ -88,15 +182,56 @@ variables.profileEditButtonSelector.addEventListener("click", () => {
   openEditForm();
 });
 
-variables.profileAddButtonSelector.addEventListener("click", () =>
-  openAddForm()
+// POPUP WITH FORM - AVATAR FORM
+
+const avatarModal = new PopupWithForm(
+  variables.avatarModal,
+  handleAvatarFormSubmit
 );
+avatarModal.setEventListeners();
+
+function openAvatarForm() {
+  avatarModal.open();
+}
+
+variables.avatarIconSelector.addEventListener("click", () => {
+  openAvatarForm();
+});
+
+function handleAvatarFormSubmit(inputValues) {
+  setButtonText(variables.avatarModalButton, "Saving...");
+  api
+    .updateAvatar(inputValues.link)
+    .then((res) => {
+      variables.avatarImage.src = inputValues.link;
```

Comment:
`      variables.avatarImage.src = inputValues.link;
`

Only class `Section` should add any card to the DOM.